lib/Service/FormsService.php

```diff
@@ -655,6 +655,7 @@ public function getFilePath(Form $form): ?string {
 	}
 
 	public function getFileName(Form $form, string $fileFormat): string {
+		/** @psalm-suppress TypeDoesNotContainType */
 		if (empty(Constants::SUPPORTED_EXPORT_FORMATS[$fileFormat])) {
```

Comment:
Instead of silence the message:

```suggestion
		if (isset(Constants::SUPPORTED_EXPORT_FORMATS[$fileFormat])) {
```

Reply:
Thank you :)